gdc_ng_models/models/misc.py

```diff
@@ -30,5 +29,8 @@ class FileReport(Base):
     report_data = Column(JSONB, nullable=True)
 
     __table_args__ = (
+        PrimaryKeyConstraint("id","timestamp"),
         Index("filereport_report_data_idx", "report_data", postgresql_using="gin",),
+        Index("timestamp_idx", "timestamp", postgresql_using="btree",),
```

Comment:
Is this line necessary? When a primary key is declared it is automatically indexed, and the default index type in PostgreSQL is b-tree.

Reply:
Thanks, this was leftover from when timestamp was not a primary key. Made the change.